samples/videoDecode/videodecode.cpp

```diff
@@ -203,18 +214,26 @@ int main(int argc, char **argv) {
     try {
         std::size_t found_file = input_file_path.find_last_of('/');
         std::cout << "info: Input file: " << input_file_path.substr(found_file + 1) << std::endl;
+        RocVideoDecoder *viddec;
```

Comment:
can we initialize to `nullptr`?

Reply:
Not really needed as it will get reassigned anyway in line#221 onwards